testsuite/cluster-test/src/instance.rs

```diff
@@ -72,6 +89,35 @@ impl Instance {
         Ok(())
     }
 
+    pub async fn run_cmd_inner_async<I, S>(&self, no_std_err: bool, args: I) -> failure::Result<()>
+    where
+        I: IntoIterator<Item = S>,
+        S: AsRef<OsStr>,
+    {
+        let ssh_dest = format!("ec2-user@{}", self.ip);
+        let ssh_args = vec![
+            "ssh",
+            "-i",
+            "/libra_rsa",
+            "-oStrictHostKeyChecking=no",
+            "-oConnectTimeout=3",
+            "-oConnectionAttempts=10",
+            ssh_dest.as_str(),
+        ];
+        let mut ssh_cmd = tokio::process::Command::new("timeout");
```

Comment:
Instead of using the `timeout` binary on linux, we can wrap the `tokio::process::Command` using a `tokio::time::Timeout` : https://docs.rs/tokio/0.2.0/tokio/time/fn.timeout.html

Reply:
I am not sure if using this timeout will actually terminate process.
Most likely will just close process handle

Reply:
I *think* you can get that behavior with [kill_on_drop](https://docs.rs/tokio/0.2.0/tokio/process/struct.Command.html#method.kill_on_drop)